### PR TITLE
Freeform article on governance

### LINKED ISSUE #173: Freeform article on governance (Additional)
# Request for content

_Be sure to check out the [Guide for submitting a new Issue](https://monthly.aragon.one/guides/guide_for_submitting_a_new_issue/) for submitting a new request_

### Request type:
_Article_ 

#### Content type:
_Opinion Piece_

### Description of the requested content
Thoughtful, freeform article on the subject governance. You have the freedom to write an editorial piece of your choosing.

### Additional information
Minimum 1500 words, maximum 2500 words

#### Deadline
By the end of the month, in time for Aragon Monthly issue #06

### PR DESCRIPTION
**Pull request by marikio**
Article for Issue #173

# New content for Aragon Monthly

_Be sure to check out the [Guide for submitting a new Pull Request](https://monthly.aragon.one/guides/guide_for_submitting_a_new_pull_request/) before submitting your PR_

### Closing of a referenced Issue
Closes #173 

### Content category
_Article_

#### Content type:
_Opinion Piece_

### Title
Self-Governance as a Human Right

### Description
This is a 2,342 word article discussing the necessity of establishing “self-governance” as a human right. The article begins with an acknowledgement of the current problems with big government. An introduction is then provided on the concept of self-governance as a human right. A brief description is given of how governance relates to economic, political, and societal spheres. Finally, an overview of the distributed governance field is provided with a description on how multiple organizations are contributing to the establishment of self-governance as a human right.

### CODE REVIEW

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+ 
+But recently there have been several important shifts. Advancements in technology have made distribution networks and infrastructure easier to build. Refugees and migrants are increasing for a variety of reasons, including global warming. And thirdly war and violence are decreasing.
+
+These conditions create an ecosystem ripe for individuals to begin experimenting with creating their own “mini societies”.
+ 
+And at first this seems like a nice little experiment, a luxury for alternative lifestyles to prosper outside of the formal state. But when we look closer at how oppressive governments and regimes have become relative to our expectations, we realize that governments are not fulfilling their part of the social contract. They are failing their citizens and non-registered residents in monumental ways.
+
+People with disabilities are legally allowed to be paid sub minimum wage. Sex workers are constantly threatened through predatory legislation and a host of legislation threatens the internet as we know it.
```
Comment (krrisis) [inline docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: twice legislation in one sentence, might want to adapt? just a detail

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+If we consider things such as life, liberty, and the pursuit of happiness to be inalienable, shouldn’t we be able to pursue them outside the context of a state? And if the state is the main culprit of oppression, shouldn’t our right to self-govern supersede all other rights?
+ 
+Self-governance is not a luxury, it is a human right.
+ 
+The movement to reframe self-governance as an inalienable human right is in its nascency. Organizations like Amnesty International and the United Nations still function on the premise of nation states, and most activists, political change makers, and human rights advocates still see the governments as not only central to their work, but the key component of change making. Identify a human rights crisis, investigate, work with lawmakers to craft legislation, and then lobby.
+ 
+In the coming decades we need to start considering alternative paths. Circumventing corrupt governments to provide opportunity for communities to self organize, self govern, and start their own societies is one approach.
```
Comment (krrisis) [inline docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: self-govern

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## Building the Future
+
+In the world of distributed governance there is work being done to create the political, economic, and societal systems of tomorrow. 
+
+The blockchain community is frustrated by centralization. We hate the repercussions of large corporations having control over our data, digital assets, and our bodies. And so, as engineers and entrepreneurs we want to contribute to a movement that diminishes the power of these oppressive institutions.
+
+But governments play just as much, if not a greater, role in centralization of power. Instead of controlling servers, governments control land, borders, and behavior through the mask of nation states.
+
+So the question becomes what can the blockchain community do to contribute to the human rights struggle for self-governance.
```
Comment (krrisis) [inline docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: question mark at the end?

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+We won’t be able to implement an effective system of distributed governance unless we take a three-pronged approach, focusing on each sphere.
+
+And fortunately, that’s how governance on the blockchain is evolving.
+
+Corporations and Organizations
+When we look at corporations and organizations we see many new blockchain startups building the infrastructure to tackle this institution. Companies like Aragon lead the way, embracing the culture and language of DAOs or Decentralized Autonomous Organizations.
+
+DAO’s are entities that rely on smart contracts, pre-programmed rules that define the behavior and allowed actions within a system.
```
Comment (krrisis) [inline docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: DAOs

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+ 
+For example, you can program smart contracts to carry out tasks such as releasing funds on a certain date or when a percentage of voters agree to fund a project. Essentially, DAOs are a way to cryptographically guarantee democracy. Participants are able to vote on new rules, change rules, remove members, and come to consensus on issues which can then trigger system action.
+
+No centralized control. No corrupt intermediaries.
+
+While Aragon has embraced the language of DAO’s, the other organizations in the decentralized governance space are essentially building variations of this same system.
```
Comment (krrisis) [inline docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: DAOs

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+They were in a tourist district, not too far from the hotels. Moms held kids and rucksacks, while dads pointed cameras out the bus windows taking shots of Seljuk buildings and mustached Dondurma vendors tricking tourists into reaching for their waffle cones.
+
+A few months earlier a former Turkish military man had shown her the site where she was headed. He had done so as a joke, not thinking that she would ever end up coming back. At the time, Kamaria didn’t either. It was only a while later that she realized it was the perfect spot to stand watch.
+
+Several hours after the bus had dropped her off, she headed towards the long pier, about a kilometer away from the closest hotel. She had to wait another 6 to 7 hours before the boats would start leaving port. Watching with binoculars pressed tightly to her eyes, she would count the number of boats to undock and later warn her contacts on the Greece coastline of how many refugees they should expect.
```
Comment (krrisis) [inline docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: maybe this is not too important but although this intro drags the reader in, I kinda miss a clearer link with the article, was this supposed to come back? Now I want to know more about Kamaria - is she working for the greek government? Maybe just me but this part made me wonder

**Comment by Smokyish** [comment]
@marikio Hi, please see the review that @krrisis left and do sign the CLA so we can continue with merging of the article!

**Comment by Smokyish** [comment]
@marikio As you haven't done any changes or signed the CLA, will need to push this further again to Issue 08.

**Comment by marikio** [comment]
Oops, haven't seen these. Will take a look

File: docs/articles/opinion/Self-Governance_as_a_Human_Right.md (hunk 6 above)
Comment (marikio) [reply docs/articles/opinion/Self-Governance_as_a_Human_Right.md]: Hmm, ok. I will try to link it back in towards the end.

**Pull request merged**